### DISCUSSION TITLE
Axios Network error after install wagmi with walletConnect in react-native

### DISCUSSION THREAD

**Discussion by shin101004** [wevm/wagmi#3568] [Q&A]
Hi, I develop dapp with react-native.

here is my package.json

```json
  "axios": "^1.6.2",
  "react-native": "0.72.8",
  "react-native-modal": "^13.0.1",
  "viem": "1.21.4",
  "wagmi": "1.4.13",
  "@walletconnect/react-native-compat": "^2.11.0",
  "@web3modal/wagmi-react-native": "^1.1.1",
  "ethers": "^5.7.2",
```

after I sign in with the wallet app, I request API with mutateAsync or mutate. it returns Axios Network Error.

![스크린샷 2024-02-06 오후 9 15 44](https://github.com/wevm/wagmi/assets/61345090/509734a2-287d-4e0a-ae8c-cd4b49ec3ca2)

It occurred in only 2 APIs which called after sign with the wallet app and the error didn't come from my server.

is there anyone who has experienced this issue?

plz help me

**Comment by tmm** [comment]
Wagmi doesn't use Axios so definitely not coming from anything Wagmi-related.

If you want someone to take a look, you should provide a minimal reproducible example.

**Reply by shin101004** [reply]
![스크린샷 2024-02-07 오전 4 03 47](https://github.com/wevm/wagmi/assets/61345090/8d2cfec7-9209-4fed-8848-07e9f3b1f127)


here is example.

After i signed in metamask, i call API somthing like post or put method. 
it returns error most of time.

before i use web3modal/wagmi, it doesn't happen.

**Reply by tmm** [reply]
this isn't a minimal reproducible example https://stackoverflow.com/help/minimal-reproducible-example also have `handleVerifySignature` isn't a Wagmi feature.
